src/ralph/ui/static/ui/main.js

```diff
@@ -1,284 +1,295 @@
-            '{{label}}</a>' +
-            '{{/years}}' +
-            '</div>' +
-            '<div class="btn-group months" data-toggle="buttons-radio">' +
-            '{{#months}}' +
-            '<a href="#" class="btn {{css_class}}" data-value="{{value}}">' +
-            '{{label}}</a>' +
-            '{{/months}}' +
-            '</div>' +
-            '</div>';
-
-    $('.daterange-form .form-actions').each(function (i, form) {
-        var $form = $(form);
-        var $start = $form.find('input[name="start"]');
-        var $end = $form.find('input[name="end"]');
-        require(['mustache'], function (Mustache) {
-            $form.prepend(Mustache.render(calendar_tmpl, calendar));
-
-            $form.find('.years a').click(function (e) {
-                var $this = $(this);
-                var start_date = parseDate($start.val());
-                start_date.setUTCFullYear($this.data('value'));
-                $start.val(formatDate(start_date));
-                if ($end) {
-                    var end_date = parseDate($end.val());
-                    end_date.setUTCFullYear($this.data('value'));
-                    $end.val(formatDate(end_date));
-                };
-            });
-            $form.find('.months a').click(function (e) {
-                var $this = $(this);
-
-                var date = parseDate($start.val());
-                date.setUTCMonth($this.data('value') - 1);
-                date.setUTCDate(1);
-                $start.val(formatDate(date));
-
-                if ($end) {
-                    date.setUTCMonth($this.data('value'));
-                    date.setUTCDate(0);
-                    $end.val(formatDate(date));
-                };
             });
         });
-    });
-    $('form.search-form').submit(function () {
-        var $form = $(this)
-        var fields = $form.find('input[value!=""],textarea,select').serialize();
-        var action = $form.attr('action') || '';
-        window.location = action + '?' + fields;
-        return false;
-    });
-    $('.close').click(function () {
-        if ($(this).attr('data-dismiss') == 'alert'){
-            $(this).parents('.alerts').filter(':first').remove();
-        }
-    });
-
-    /* Make the autocomplete fields show their dropdowns immediatelly. */
-    $('input.dropdown').focus(function () {
-        var t = $(this).data('typeahead');
-        var items = $.isFunction(t.source) ? this.source(t.query, $.proxy(t.process, this)) : t.source;
-        t.query = $(this).val();
-        t.process(items);
-        t.show();
+        $('form.search-form').submit(function () {
+            var $form = $(this)
+            var fields = $form.find('input[value!=""],textarea,select').serialize();
+            var action = $form.attr('action') || '';
+            window.location = action + '?' + fields;
+            return false;
+        });
+        $('.close').click(function () {
+            if ($(this).attr('data-dismiss') == 'alert'){
```

Comment:
what if not alert? In my opinion everything with class close should be clickable and do some actions.